theano/gof/cmodule.py

```diff
@@ -1467,7 +1467,7 @@ def std_lib_dirs_and_libs():
                 os.path.join(sys.prefix, 'libs'),
                 # Used in newer Canopy
                 os.path.join(sys.prefix,
-                             'EGG-INFO\mingw\usr\x86_64-w64-mingw32\lib')]
+                             'EGG-INFO\mingw\\usr\\x86_64-w64-mingw32\lib')]
```

Comment:
There are still unescaped backslashes.  Also it is possible to use forward slashes in windows paths from python.  Or to use raw strings to avoid escaping backslashes.